libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -511,22 +519,23 @@ public void setMinPasscodeLength(int minPasscodeLength) {
              * triggered later from OAuthWebviewHelper.
              */
             if (hasStoredPasscode(SalesforceSDKManager.getInstance().getAppContext())) {
-        		showLockActivity(SalesforceSDKManager.getInstance().getAppContext(),
-        				true);
+                setEnabled(true);
```

Comment:
Why do we need `setEnabled(true)` here? If it enters this code, passcode is stored and hence it is already enabled right?